templates/macros/widgets.html.twig

```diff
@@ -369,7 +369,7 @@
 {% macro tag_list(taglist) %}
     {% import _self as macro %}
     {% for tag in taglist %}
-        {{ macro.badge(tag.name , 'green') }}
+        {{ macro.badge(tag.name , tag.color ?? 'red') }}
```

Comment:
'green' is not 'red' 😁 

And we should not rely on the pre-defined browser colors, but use the color codes from the css. 
bg-green: `#00a65a`
bg-red: `#dd4b39`

So: `tag.color ?? '#00a65a'`

Reply:
Implemented the suggestion below:
`{{ color|default('##00a65a') }}`
So this isn't needed anymore, right?

Reply:
Please push your changes, so I can see them

Reply:
Done ;)

Reply:
With the latest push you still change the default color from red to green. We could discuss whether red is the correct default color for a badge ... but then we have to check all usages of the macro (not only in core but also in plugins).

Badges are not only used in combination with tags. Please change the default color for tags to `#dd4b39` and the default for tags as you had it before to `tags.color|default('#00a65a')`

Reply:
Now I got it, sorry. ;)